README.md

```diff
@@ -25,7 +25,7 @@ jupyter nbextension enable --py [--sys-prefix|--user|--system] nbsv
 
 Create a dev environment:
 ```bash
-conda create -n nbsv-d -c conda-forge yarn python jupyterlab=3.6.4
+conda create -n nbsv -c conda-forge yarn python jupyterlab=3.6.4
```

Comment:
Not sure that this is a typo, but also doesn't really matter whether you name the environment `nbsv` or `nbsv-d` as long as any instructions are consistent.